lib/node_modules/@stdlib/iter/cuany-by/examples/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+var randu = require( '@stdlib/random/iter/randu' );
+var iterMap = require( '@stdlib/iter/map' );
+var iterCuAnyBy = require( './../lib' );
+
+function threshold( r ) {
+	return ( r > 0.95 );
+}
+
+// Create an iterator which generates uniformly distributed pseudorandom numbers:
+var opts = {
+	'iter': 100
+};
+var riter = randu( opts );
+
+// Create an iterator which applies a threshold to generated numbers:
+var miter = iterMap( riter, threshold );
+
+// Create an iterator which cumulatively tests whether at least one iterated value is truthy:
+var it = iterCuAnyBy( miter );
```

Comment:
Your example is missing a predicate function. You don't need `miter` here. You could pass `threshold` directly to `iterCuAnyBy`.